libmscore/instrument.h

```diff
@@ -292,6 +294,12 @@ class Instrument {
       QString instrumentId()                                 { return _instrumentId; }
       void setInstrumentId(const QString& instrumentId)      { _instrumentId = instrumentId; }
 
+      void setStaffGroup(const StaffGroup staffGroup)        { _staffGroup = staffGroup; }
+      const StaffGroup staffGroup() const                    { return _staffGroup; }
+      StaffGroup staffGroup()                                { return _staffGroup; }
+      void setLines(int lines)                               { _lines = lines; }
+      const int lines() const                                { return _lines; }
+      int lines()                                            { return _lines; }
```

Comment:
This should be `int lines() const`, since we're not modifying any values of `Instrument`.